--- libgnomecups-0.2.3/libgnomecups/gnome-cups-i18n.c.orig	2007-01-02 01:18:51.000000000 +0100
+++ libgnomecups-0.2.3/libgnomecups/gnome-cups-i18n.c	2013-01-06 08:19:49.888606151 +0100
@@ -1,5 +1,5 @@
 #include <config.h>
-#include <glib/gmacros.h>
+#include <glib.h>
 #include "gnome-cups-i18n.h"
 
 #ifdef ENABLE_NLS
--- libgnomecups-0.2.3/libgnomecups/gnome-cups-init.h.orig	2007-01-02 01:18:51.000000000 +0100
+++ libgnomecups-0.2.3/libgnomecups/gnome-cups-init.h	2013-01-06 08:20:30.967196805 +0100
@@ -1,8 +1,7 @@
 #ifndef GNOME_CUPS_INIT
 #define GNOME_CUPS_INIT
 
-#include <glib/gtypes.h>
-#include <glib/gmacros.h>
+#include <glib.h>
 
 G_BEGIN_DECLS
 
