--- libgnomecups-0.1.11/configure.in.orig	2004-08-19 20:54:05.000000000 +0200
+++ libgnomecups-0.1.11/configure.in	2004-08-24 22:57:55.394920712 +0200
@@ -39,7 +39,7 @@
 AC_PROG_INTLTOOL([0.20])
 
 dnl Add the languages which your application supports here.
-ALL_LINGUAS=" ca cs de en_CA en_GB es eu fr hr hu is it ja nb nl no pt pt_BR sr sr@Latn sv uk"
+ALL_LINGUAS=" ca cs de en_CA en_GB es eu fr hr hu is it ja nb nl pt pt_BR sr sr@Latn sv uk"
 AM_GLIB_GNU_GETTEXT
 
 AC_OUTPUT([
